maps/interiors/pizza_van.dmm

```diff
@@ -0,0 +1,246 @@
+	icon_state = "roof_4"
+	},
+/turf/open/void/vehicle,
+/area/space)
+"n" = (
+/obj/structure/interior_wall/pizza_van{
+	icon_state = "background_2";
+	pixel_y = 32;
+	layer = 5.2;
+	alpha = 50
+	},
+/turf/open/void/vehicle,
+/area/space)
+"o" = (
+/obj/effect/landmark/interior/spawn/interior_viewport/simple{
+	pixel_y = 30
+	},
+/obj/structure/bed/chair/comfy/beige,
+/obj/item/clothing/head/hybrisa/pizza_galaxy,
+/obj/effect/vehicle_roof/pizza_van{
+	icon_state = "roof_3"
+	},
+/turf/open/shuttle/vehicle/med{
+	color = "#9c9a97"
+	},
+/area/interior/vehicle/pizza_van)
+"r" = (
+/obj/structure/interior_wall/pizza_van{
+	icon_state = "exterior_3"
+	},
+/obj/effect/vehicle_roof/pizza_van{
+	icon_state = "roof_5"
+	},
+/turf/open/void/vehicle,
+/area/space)
+"s" = (
+/obj/structure/interior_wall/pizza_van{
+	icon_state = "back_2"
+	},
+/turf/open/void/vehicle,
+/area/space)
+"t" = (
+/obj/effect/landmark/interior/spawn/entrance{
+	dir = 1;
+	exit_type = /obj/structure/interior_exit/vehicle/pizza_van/left;
+	name = "left entrance marker";
+	pixel_y = 32;
+	tag = "left"
+	},
+/obj/effect/landmark/interior/spawn/vehicle_driver_seat{
+	dir = 4;
+	icon_state = "comfychair_beige"
+	},
+/obj/effect/landmark/interior/spawn/interior_viewport/simple/windshield{
+	icon_state = "windshield_viewport_bottom";
+	icon = 'icons/obj/vehicles/interiors/pizza_van_interior.dmi'
+	},
+/obj/effect/vehicle_roof/pizza_van{
+	icon_state = "front_roof_2"
+	},
+/turf/open/shuttle/vehicle{
+	icon_state = "floor_3_11"
+	},
+/area/interior/vehicle/pizza_van)
+"v" = (
+/obj/effect/landmark/interior/spawn/interior_viewport/simple/windshield{
+	icon = 'icons/obj/vehicles/interiors/pizza_van_interior.dmi'
+	},
+/obj/effect/vehicle_roof/pizza_van{
+	icon_state = "interior_door"
+	},
+/obj/effect/vehicle_roof/pizza_van{
+	icon_state = "front_roof_1"
+	},
+/turf/open/void/vehicle,
+/area/space)
+"w" = (
+/obj/effect/vehicle_roof/pizza_van{
+	icon_state = "front_3"
+	},
+/turf/open/void/vehicle,
+/area/space)
+"y" = (
+/obj/effect/landmark/interior/spawn/entrance{
+	alpha = 50;
+	exit_type = /obj/structure/interior_exit/vehicle/pizza_van/right;
+	name = "right entrance marker";
+	tag = "right"
+	},
+/obj/structure/bed/chair/comfy/beige{
+	dir = 4
+	},
+/turf/open/shuttle/vehicle{
+	icon_state = "floor_3_12"
+	},
+/area/interior/vehicle/pizza_van)
+"G" = (
+/obj/effect/landmark/interior/spawn/interior_viewport/simple{
+	alpha = 100;
+	pixel_y = -2
+	},
+/obj/structure/bed/chair/comfy/beige{
+	dir = 1
+	},
+/obj/item/stack/sheet/cardboard{
+	layer = 1;
+	pixel_x = 12
+	},
+/turf/open/shuttle/vehicle/med{
+	color = "#9c9a97"
+	},
+/area/interior/vehicle/pizza_van)
+"J" = (
+/obj/structure/interior_wall/pizza_van{
+	icon_state = "background_3";
+	pixel_y = 32;
+	layer = 5.2;
+	alpha = 50
+	},
+/turf/open/void/vehicle,
+/area/space)
+"M" = (
+/obj/effect/landmark/interior/spawn/entrance{
+	dir = 8;
+	exit_type = /obj/structure/interior_exit/vehicle/pizza/backleft;
```

Comment:
This type doesn't exist:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/ca12c2d2-66b1-4f51-a6a0-9ae374730ff0)

Reply:
should be fixed now. instance misspelling.